html/includes/table/devices.inc.php

```diff
@@ -207,9 +207,13 @@
         $location = substr($device['location'], 0, 32);
     }
 
+    if ($device['features']) {
```

Comment:
You can just wrap device features in brackets inline below rather than setting the variable with it.

Reply:
It simply follows how it's done in other files. The argument is null often so then it would be an empty `()`

Reply:
Ternary :)